index.html

```diff
@@ -7,6 +7,11 @@
 		<meta name="description" content="Decentralized Marketplaces" />
 		<link rel="apple-touch-icon" href="/icon-192x192.png" />
 		<link rel="manifest" href="/manifest.json" />
+
+		<meta name="msapplication-TileColor" content="#da532c">
+        <meta name="theme-color" content="#ffffff">
```

Comment:
Created #36 for this, but we can fix it later.

Reply:
Can we mark this as resolved?